src/model.test.ts

```diff
@@ -0,0 +1,123 @@
+import * as _ from 'lodash';
+import { Request } from 'express';
+import { HubApiModel } from '../src/model';
+import { PassThrough, pipeline } from 'stream';
+import { promisify } from 'util';
+import axios from 'axios';
+jest.mock('axios');
+
+describe('HubApiModel', () => {
+  it('streams hub search ogc items', async () => {
+    const mockedResponse = {
+      data: {
+        features: [{
+          id: 'a123',
+          type: 'Feature',
+          properties: {
+            title: 'item-title'
+          }
+        }],
+        links: [
+          {
+            rel: 'prev'
+          }
+        ]
+      }
+    };
+    (axios.get as jest.Mock).mockResolvedValue(mockedResponse);
+    const model = new HubApiModel();
+
+    const req = {
+      res: {
+        locals: {
+          siteIdentifier: 'https://my-site.hub.arcgis.com'
```

Comment:
Hold on, I've seen `siteIdentifier` either be a URL OR a hostname in different test files. Which should it be? We should rename this variable accordingly so future developers also know.

Reply:
Apologies for the confusion `siteIdentifier` is hub site url. I have fixed the read me example.